frontend/app/src/components/helper/Card.vue

```diff
@@ -7,11 +7,11 @@
     }"
     v-on="rootListeners"
   >
-    <v-card-title v-if="$slots.title">
+    <v-card-title v-if="$slots.title" :class="{ 'pt-6': $slots.icon }">
```

Comment:
also no need for action right now but we probably need to start migrating to `useSlots`